From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Fri, 5 Dec 2025 15:34:17 +0900
Subject: No need to use tzdata module on Unix

Forwarded: https://github.com/kovidgoyal/calibre/pull/2920

Windows uses tzdata module for zoneinfo, but Unix uses system's zoneinfo
data.

See also:
  https://docs.python.org/3/library/zoneinfo.html#module-zoneinfo
---
 pyproject.toml            | 1 -
 src/calibre/test_build.py | 3 +--
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/pyproject.toml b/pyproject.toml
index fd842ef..2c7bbe8 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -54,7 +54,6 @@ dependencies = [
 	"PyQt6_WebEngine == 6.10.0",
 	"MacFSEvents == 0.8.4; sys_platform == 'darwin'",
 	"xxhash == 3.6.0",
-    "tzdata == 2025.2",
     "tzlocal == 5.3.1",
 ]
 
diff --git a/src/calibre/test_build.py b/src/calibre/test_build.py
index 156a870..bc0b03f 100644
--- a/src/calibre/test_build.py
+++ b/src/calibre/test_build.py
@@ -86,9 +86,8 @@ class BuildTest(unittest.TestCase):
         del CHMFile, chmlib
 
     def test_tzdata(self):
-        import tzdata
         import tzlocal
-        del tzlocal, tzdata
+        del tzlocal
 
     def test_chardet(self):
         from calibre_extensions.uchardet import detect
